code/web/src/modules/user/Profile.js

```diff
@@ -9,54 +9,233 @@ import { Link } from 'react-router-dom'
-
-        <Button theme="secondary" onClick={props.logout} style={{ marginLeft: '1em' }}>Logout</Button>
-      </GridCell>
-    </Grid>
-  </div>
-)
+class Profile extends PureComponent {
+
+  constructor() {
+    super()
+  }
+
+  // Runs on server only for SSR
+  static fetchData({ store }) {
+    return store.dispatch(getListByUser())
+  }
+
+  // Runs on client only
+  componentDidMount() {
+    this.props.getListByUser()
+  }
+
+  render() {
+    const products = mockProductHistory.map((product, i) => {
+      return (
+        <GridCell key={i}>
+          <Card style={{ width: '18em', backgroundColor: white, margin: 'auto' }}>
+            <p style={{ padding: '2em 3em 0 3em' }}>
+              <img src={routeImage + product.image} alt={product.slug} style={{ width: '100%' }}/>
+            </p>
+
+            <div style={{ padding: '1em 1.2em' }}>
+              <H4 font="secondary" style={{ color: black }}>{ product.name }</H4>
+              <p>Description: {product.description}</p>
+              <p>Order Number: {product.createdAt}</p>
+              <p>Product Status: {product.kept ? "Kept" : "Returned"}</p>
+            </div>
+          </Card>
+        </GridCell>
+      )
+    })
+
+    return (
+      <div>
+        {/* SEO */}
+        <Helmet>
+          <title>My Profile - Crate</title>
+        </Helmet>
+
+        {/* Top title bar */}
+        <Grid style={{ backgroundColor: grey }}>
+          <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+            <H3 font="secondary">My profile</H3>
+          </GridCell>
+        </Grid>
+
+        {/* User information */}
+        <Grid justifyCenter={true} alignCenter={true}>
+          <GridCell style={{
+              marginLeft: '2em',
+              padding: '2em',
+              textAlign: 'center',
+              objectFit: 'contain',
+              maxWidth: 250,
+              maxHeight: 250
+            }}>
+            <ImageTile height={250} width={250} image={userImage} shadow={level3}/>
+          </GridCell>
+
+          <GridCell>
+            <H4 style={{ marginBottom: '0.5em' }}>{this.props.user.details.name}</H4>
+
+            <p style={{ color: grey2, marginBottom: '2em' }}>User Description: Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia, molestiae quas vel sint commodi repudiandae consequuntur voluptatum laborum numquam blanditiis harum quisquam eius sed odit fugiat iusto fuga praesentium optio</p>
+            <p style={{ color: grey2, marginBottom: '2em' }}>Email Address: {this.props.user.details.email}</p>
+            <p style={{ color: grey2, marginBottom: '2em' }}>Shipping Address: 123 Main Street, Denver, CO 80602</p>
+
+          </GridCell>
+        </Grid>
+
+        {/* Buttons */}
+        <Grid>
+          <GridCell style={{ flex: '' , textAlign: 'center', marginBottom: '2em' }}>
+            <Button theme="secondary" onClick={() => console.log('wow!')} style={{ marginRight: '1em' }}>Update Profile</Button>
```

Comment:
Love that we didn't have to create any components here, just using the button and card styling that already exists!